src/backend.github.js

```diff
@@ -324,6 +441,31 @@ var _ = Mavo.Backend.register($.Class({
 			});
 	},
 
+	switchToMyForkDialog: function(forkURL) { 
+			let params = (new URL(location)).searchParams;
+			params.append("storage", forkURL + "/" + this.path);
+
+			this.notice = this.mavo.message(`
+			${this.mavo._("gh-login-fork-options")}
+			<form onsubmit="return false">
+				<a href="${location.pathname}?${params}"><button>${this.mavo._("gh-use-my-fork")}</button></a>
+			</form>`, {
+				classes: "mv-inline",
+				dismiss: ["button", "submit"]
+			});
+
+			this.notice.closed.then(form => {
+				if (!form) {
+					return;
+				} 
+
+				history.pushState({}, "", `${location.pathname}?${params}`);
+				location.replace(`${location.pathname}?${params}`); 
```

Comment:
Have you verified this doesn't get triggered if the notice is just dismissed?

Reply:
Yes